sdk/core/abort-controller/rollup.base.config.js

```diff
@@ -66,29 +66,24 @@ export function nodeConfig(test = false) {
 export function browserConfig(test = false) {
```

Comment:
This is just a cleanup, removing some useless configuration that I imagine was copy/pasted from the template.